config/theme.toml

```diff
@@ -38,12 +38,16 @@ bold = true
 
 # Selected files (standard selection)
 [selection]
-fg = "light_yellow"
+fg = "light_green"
 bold = true
 
 # Files selected in current visual mode
 [visual_mode_selection]
-fg = "light_red"
+fg = "light_yellow"
+bold = true
+
+[mark]
```

Comment:
Wouldn't it be nice to have different highlight groups for different file operations? Maybe something like `[mark.copy]`, `[mark.cut]` and `[mark.link]`